discovery/src/main/scala/com.wavesplatform.discovery/network/MessageCodec.scala

```diff
@@ -0,0 +1,39 @@
+package com.wavesplatform.discovery.network
```

Comment:
I'm sure you don't need to encode all of this, either reiuse if it's not important, or clean this from Microblocks, blocks, etc